src/main/java/org/concordion/api/AbstractCommand.java

```diff
@@ -25,7 +25,4 @@ public boolean isExample() {
 
     public void executeAsExample(CommandCall commandCall, Evaluator evaluator, ResultRecorder resultRecorder) {
     }
-
-    public void finish(CommandCall commandCall) {

```

Comment:
Given that this method is new in 2.0.0 and we have only published a SNAPSHOT, I'm not too fussed about the potential breakage.